src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java

```diff
@@ -50,7 +50,16 @@ public void updateCard(Long id, Card card) {
         return cardsQueue;
     }
 
-    public void deleteCard(Long id) {
-        cardRepository.delete(id);
+    public void deleteCard(Long deck_id, Long card_id) {
```

Comment:
Does it work without @ Transactional annotation?

Reply:
Yes, it works without @ Transactional annotation